services/src/main/java/org/keycloak/email/DefaultEmailSenderProvider.java

```diff
@@ -111,7 +112,8 @@ public void send(RealmModel realm, UserModel user, String subject, String textBo
             Message msg = new MimeMessage(session);
             msg.setFrom(new InternetAddress(from));
             msg.setHeader("To", address);
-            msg.setSubject(subject);
+            msg.setSubject(MimeUtility.encodeText(subject, "utf-8", "B"));
```

Comment:
As i am understand documentation ( https://javamail.java.net/nonav/docs/JavaMail-1.5.pdf ), we should use `MimeMessage` instead of `Message`. It will correct setup subject with `msg.setSubject(subject,"utf-8")` against `setSubject(MimeUtility.encodeText(subject,"utf-8",(String)null )`. Cause `MimeMessage` internally encoding anyway, when we use `msg.setSubject(subject)`.